tests/ViewRendererTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+namespace Tests;
+
+
+use Illuminate\View\Compilers\BladeCompiler;
+use Illuminate\View\Factory;
+use TightenCo\Jigsaw\View\ViewRenderer;
+
+class ViewRendererTest extends TestCase
+{
+    public function testItRegisteresHintPaths()
+    {
+        $mock = \Mockery::mock(Factory::class);
+        $mock->shouldReceive('getFinder');
+        $mock->shouldReceive('addNamespace')->with('view::hint', 'path');
+        $mock->shouldReceive('addExtension');
+        new ViewRenderer($mock, \Mockery::mock(BladeCompiler::class), [
+            'viewHintPaths' => [
+                'view::hint' => 'path'
+            ]
+        ]);
+        $this->addToAssertionCount(
+            \Mockery::getContainer()->mockery_getExpectationCount()
+        );
+        \Mockery::close();
+    }
```

Comment:
Throughout this test, can you update to follow style conventions from other tests?

- annotate with `@test`, and camel case test names:

```
    /**
     * @test
     */
    public function it_registers_hint_paths()
    {
```

- import `Mockery` with a `use` statement, and remove the trailing `\`
- no need to call `Mockery::close()`, that's handled in `tearDown()`